package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "aws-sdk": "^2.172.0",
-    "bluebird": "^3.5.0",
+    "bluebird": "^3.5.1",
```

Comment:
I'm pretty sure we're not actually using bluebird at all...  It can probably be deleted.